docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -278,35 +278,18 @@ For more information, see xref:../troubleshooting/troubleshooting-driver-plugins
 
 . Verify that you pair the virtual switch with a network card (wired or wireless) that is connected to the network.
 
-. Set the environment variable `HYPERV_VIRTUAL_SWITCH` to the name of the external virtual switch you want to use for {project}.
+. Set the configuration option `hyperv-virtual-switch` to the name of the external virtual switch you want to use for {project}.
 +
-For example, on Command Prompt use:
+For example, on PowerShell use
 +
 ----
-C:\> set HYPERV_VIRTUAL_SWITCH=External (Wireless)
+PS> minishift config set hyperv-virtual-switch "External (Wireless)"
```

Comment:
We should mention about the `--hyperv-virtual-switch` flag too.

Reply:
added